cmd/bbctl/desktopauth.go

```
@@ -0,0 +1,205 @@
+
+func readDesktopAccount(dbPath string) (*DesktopAccount, error) {
+	dbURI := (&url.URL{
+		Scheme:   "file",
+		Path:     dbPath,
+		RawQuery: "mode=ro",
+	}).String()
+	db, err := sql.Open("sqlite3", dbURI)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open desktop account database: %w", err)
+	}
+	defer db.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `db.Close` is not checked

(errcheck)